azure-pipelines.yml

```diff
@@ -50,7 +50,7 @@ stages:
         - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
           # DotNet-Blob-Feed provides: dotnetfeed-storage-access-key-1
           # DotNet-Symbol-Server-Pats provides: microsoft-symbol-server-pat, symweb-symbol-server-pat
-          # Publish-Build-Assets provides: MaestroAccessToken, BotAccount-dotnet-maestro-bot-PAT
+          # Publish-Build-Ass['=-ets provides: MaestroAccessToken, BotAccount-dotnet-maestro-bot-PAT
```

Comment:
Typo?  If not typo, maybe add comment why this is here.

Reply:
Definitely a typo.